import/src/test/java/com/google/cloud/healthcare/imaging/dicomadapter/CStoreServiceTest.java

```diff
@@ -308,14 +327,16 @@ private void basicCStoreServiceTest(
       int expectedDimseStatus,
       MockDestinationConfig[] destinationConfigs,
       String sopClassUID,
-      String sopInstanceUID) throws Exception {
+      String sopInstanceUID,
+      String testFile,
```

Comment:
testFile not used?

Reply:
Right... I've been applying transcoding to default .dcm file, which contains no single frame. Which technically passed.

Fixed this.